app/src/test/java/com/courseproject/tindar/controllers/userlist/UserListControllerUnitTest.java

```diff
@@ -0,0 +1,25 @@
+package com.courseproject.tindar.controllers.userlist;
+
+import com.courseproject.tindar.usecases.userlist.UserListInputBoundary;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class UserListControllerUnitTest {
```

Comment:
This isn't controller unit test. You need to input mockUserListUserInput as an argument when instantiating controller, and call the method in the controller.